src/routes/(site)/[...number]/+page.server.ts

```diff
@@ -0,0 +1,17 @@
+import { prisma_client } from '$/server/prisma-client';
+import get_show_path from '$/utilities/slug';
+import { error, redirect } from '@sveltejs/kit';
+
+export async function load(event) {
```

Comment:
Extract this into `lib` so it can be re-used in both spots without copy / paste.

Reply:
Going to mark as resolved, but please re-open if you aren't cool with that.